webcompat/templates/form.html

```diff
@@ -121,7 +121,7 @@ <h1 class="headline-1">Report Site Issue</h1>
       </div>
       {{ form.contact(class_='form-field text-field',
          placeholder='GitHub nickname',
-         pattern='^[a-z\d](?:[a-z\d]|-(?=[a-z\d])){0,38}$') }}
+         pattern='^[a-z\\d](?:[a-z\\d]|-(?=[a-z\\d])){0,38}$') }}
```

Comment:
This seems kinda unrelated to tests, but thanks for fixing.

Reply:
this was created a deprecationwarning in jinja. that pytest was exposing.

Reply:
```
====== warnings summary =======
tests/unit/test_urls.py::TestURLs::test_extracted_ga_params_end_up_as_inline_js
  /Users/karl/code/webcompat.com/env/lib/python3.7/site-packages/jinja2/lexer.py:648: DeprecationWarning: invalid escape sequence '\d'
    .decode("unicode-escape")

-- Docs: https://docs.pytest.org/en/latest/warnings.html
```